### DISCUSSION TITLE
Support for "Enums" in Sqlite

### DISCUSSION THREAD

**Discussion by Sillyvan** [SeaQL/sea-orm#2394] [Ideas]
First up im well aware that Sqlite dosent have an enum type!

Dosent mean its impossible to have something similar. A good example i have is Drizzle which makes this work.
https://orm.drizzle.team/docs/column-types/sqlite
https://github.com/drizzle-team/drizzle-orm/blob/main/drizzle-orm/src/sqlite-core/columns/text.ts

Reason for this feature is mostly because migrations fail if you decide to move from postgres to sqlite while using enums. But it would be a QoL thing regardless. Or is this for some reason impossible here?

**Comment by 0-don** [comment]
cool idea
